### PR TITLE
The HICS Etna

### PR DESCRIPTION
**Pull request by lavillastrangiato**
* Adds the HICS Etna, a Hephaestus space station orbiting Burszia I. Mapped mostly by Robotic Potato, IIRC, edited and converted to 3/4ths by me, ghostroles by RustingWithYou.

This will be test-merged.

### CODE REVIEW

**Comment by lavillastrangiato** [comment]
!review

File: code/game/objects/structures/ECD.dm [hunk 1]
```diff
@@ -0,0 +1,83 @@
+#define ECD_LOOSE 0
+#define ECD_BOLTED 1
+#define ECD_WELDED 2
+
+/obj/structure/ecd
+	name = "Electronic Countermeasures Device"
```
Comment (NonQueueingMatt) [inline code/game/objects/structures/ECD.dm:6]: lowercase names, this is not a proper noun

File: code/game/objects/structures/ECD.dm [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	. = ..()
+	switch(state)
+		if(ECD_LOOSE)
+			to_chat(user, SPAN_NOTICE("\The [src] isn't attached to anything."))
+		if(ECD_BOLTED)
+			to_chat(user, SPAN_NOTICE("\The [src] is bolted to the floor."))
+		if(ECD_WELDED)
+			to_chat(user, SPAN_NOTICE("\The [src] is bolted and welded to the floor."))
+	if(user.isSynthetic())
+		to_chat(user, SPAN_NOTICE("\The [src] does not seem to be doing anything, but you can feel it. A signal, beyond anything you can consciously understand, weaving and scratching a shield around the back of your mind."))
```
Comment (NonQueueingMatt) [inline code/game/objects/structures/ECD.dm:25]: maybe use a more menacing SPAN than notice

File: code/game/objects/structures/ECD.dm [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+/obj/structure/ecd/attackby(obj/item/W, mob/user)
+	if(W.iswrench())
+		switch(state)
+			if(ECD_LOOSE)
+				state = ECD_BOLTED
+				playsound(get_turf(src), W.usesound, 75, TRUE)
```
Comment (NonQueueingMatt) [inline code/game/objects/structures/ECD.dm:32]: use_tool

File: code/game/objects/structures/ECD.dm [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+				user.visible_message(SPAN_NOTICE("\The [user] secures \the [src] to the floor."), \
+					SPAN_NOTICE("You secure \the [src]'s external reinforcing bolts to the floor."), \
+					SPAN_WARNING("You hear a ratcheting noise."))
+				anchored = TRUE
+			if(ECD_BOLTED)
+				state = ECD_LOOSE
+				playsound(get_turf(src), W.usesound, 75, TRUE)
```
Comment (NonQueueingMatt) [inline code/game/objects/structures/ECD.dm:39]: use_tool

File: maps/event/burzsia_station/burzsia_station.dm [hunk 5]
```diff
@@ -0,0 +1,30 @@
+/datum/map/event/burzsia_station
+	name = "HICS Etna"
+	full_name = "HICS Etna"
+	path = "event/burzsia_station"
+	lobby_icons = list('icons/misc/titlescreens/sccv_horizon/sccv_horizon.dmi', 'icons/misc/titlescreens/aurora/synthetics.dmi', 'icons/misc/titlescreens/aurora/tajara.dmi', 'icons/misc/titlescreens/aurora/vaurca.dmi')
+	lobby_transitions = FALSE
+
+	allowed_jobs = list(/datum/job/visitor, /datum/job/xo, /datum/job/captain, /datum/job/chief_engineer, /datum/job/consular, /datum/job/cmo, /datum/job/hos, /datum/job/representative, /datum/job/operations_manager, /datum/job/rd)
```
Comment (NonQueueingMatt) [inline maps/event/burzsia_station/burzsia_station.dm:8]: are the head staff jobs being here intentional? like are these for horizon crew?

Comment (lavillastrangiato) [reply maps/event/burzsia_station/burzsia_station.dm:8]: dm'd

**Review by Geevies — APPROVED** [review]

File: code/game/objects/structures/ECD.dm [hunk 6]
```diff
@@ -0,0 +1,83 @@
+#define ECD_LOOSE 0
+#define ECD_BOLTED 1
+#define ECD_WELDED 2
+
+/obj/structure/ecd
```
Comment (FluffyGhoster) [inline code/game/objects/structures/ECD.dm:5]: What is the point of this object? It doesn't seem to do anything except showing messages / playing sounds for what I'm seeing

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request closed without merging**